### ISSUE TITLE
What is a complex shape?

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#2050]
The definition for perimeter is:

> Perimeter
> continuous line forming the boundary of a shape not including shared pixels, or the minimum bounding box of a complex shape, whichever is shortest. For example, the perimeter calculation for a rectangle is 2h+2w -4, where h is the height and w is the width and the corners are not counted twice. The perimeter of a circle is 2𝜋r.

I suggest that we remove "complex". A circle isn't complex, but what about a triangle/pentagon/dodecahedron? We have no ability to define complex so should just remove this term. Alternatively, someone should come up with a definition for complex (good luck!).

**Comment by alastc** [comment]
I think the idea was that, for HTML/CSS at least, anything not a rectangle is 'complex'. If something _is_ rectangular then the bounding box should be the same as the perimeter. 

I agree it's not needed here though.
